src/algorithms/otsu.jl

```diff
@@ -1,6 +1,6 @@
 @doc raw"""
 ```
-t = find_threshold(Otsu(), histogram, edges)
+t = find_threshold(histogram, edges, Otsu())
```

Comment:
Sorry I missed this. Maybe we should also mention the image version here:

```suggestion
t = find_threshold(histogram, edges, Otsu())
t = find_threshold(img, Otsu(); nbins=256)
```